pages/index.js

```diff
@@ -1,9 +1,55 @@
-import AdListPage from "./AdListPage";
+import styled from "styled-components";
+import Link from "next/link";
+import Image from "next/image";
 
 export default function Home() {
   return (
     <>
```

Comment:
You do not need the fragments here as you are only rendering one child (`main`).